docs/index.md

```diff
@@ -6,7 +6,7 @@ Open Data Services Sphinx Base
 
 Table without horizonal scrollbar:
 
-```eval_rst
```

Comment:
@robredpath This show's the change necessary to convert the docs from recommonmark to MyST-Parser, which is easy to do with a find/replace.